frontend/src/game/play/play.js

```diff
@@ -7,23 +7,27 @@ const config = require('config/config.js');
 const backend = config.servers["backend"];
 
 module.exports = {
-    playCard(rownum, cardindex, target) {
-        const playData = {
-            index: cardindex,
-            row: rownum,
-            target: target,
-        };
-        document.querySelector('.card[index="'+cardindex+'"]').style.background = "yellow";
+    playCard() {
```

Comment:
I don't like this function "magically" getting its info from somewhere that isn't even in the same file (`status.clickedCard`).

It should take the clicked card as an argument or it will grow unmantainable in the future, as this project grows.